projects/pastanaga/src/lib/textfield/textfield.common.ts

```diff
@@ -16,22 +16,22 @@ export class TextfieldCommon implements ControlValueAccessor, OnInit, Validator
     @Input() placeholder?: string;
     @Input() help?: string;
     @Input()
-    get required(): boolean { return this._required; }
-    set required(value: boolean) { this._required = coerceBooleanProperty(value); }
+    get isRequired(): boolean { return this._required; }
+    set isRequired(value: boolean) { this._required = coerceBooleanProperty(value); }
     protected _required = false;
     @Input() pattern?: RegExp;
     @Input() min?: number;
     @Input() max?: number;
     @Input()
-    get disabled(): boolean { return this._disabled; }
-    set disabled(value: boolean) { this._disabled = coerceBooleanProperty(value); }
+    get isDisabled(): boolean { return this._disabled; }
+    set isDisabled(value: boolean) { this._disabled = coerceBooleanProperty(value); }
     protected _disabled = false;
     @Input() isReadOnly = false;
     @Input() isLabelHidden = false;
-
+    @Input() isPlaceholderShown = false;
     @Output() valueChange: EventEmitter<any> = new EventEmitter();
     @Output() keyUp: EventEmitter<any> = new EventEmitter();
-    @Output() enter: EventEmitter<{event: KeyboardEvent, value: string}> = new EventEmitter();
+    @Output() enter: EventEmitter<any> = new EventEmitter();
```

Comment:
I don't like to send `any`. What are you really sending here?

Reply:
good catch, i think i changed it by mistake let me fix it